drift_flutter/pubspec.yaml

```diff
@@ -4,7 +4,6 @@ version: 0.2.5
 repository: https://github.com/simolus3/drift
 homepage: https://drift.simonbinder.eu/
 issue_tracker: https://github.com/simolus3/drift/issues
-resolution: workspace
```

Comment:
Did you run into issues without this? I think the package should still be part of the workspace.